furms-model-api/src/main/java/io/imunity/furms/api/authz/AuthzService.java

```diff
@@ -12,6 +12,7 @@
 import java.util.Set;
 
 public interface AuthzService {
+	Map<String, Object> getAttributes();
```

Comment:
I'm letting that in but surely we don't want Object typed (== untyped) attribute values :-)